src/interface.jl

```diff
@@ -165,6 +168,13 @@ function solve(
     end
     vertex_storage = FrankWolfe.DeletedVertexStorage(typeof(v)[], 1)
 
+    if use_DICG_warm_start
+        # For DICG, we use PrecomputedSet instead of ActiveSet to store information.
+        pre_computed_set = [v]
+    else
+        pre_computed_set = nothing
+    end
+
```

Comment:
You can write this in a single line as
`pre_computed_set = use_DICG_warm_start ? [v] : nothing`